.vscode/settings.json

```diff
@@ -7,7 +7,7 @@
     "**/lib": true,
     "**/dist": true
   },
-  "editor.formatOnSave": true,
+  // "editor.formatOnSave": true,
```

Comment:
Nit: remove

Reply:
oops, sorry I keep having to do this or else VsCode does bad things to files, or takes forever to save large files as clang seems to take a long time on them.